Tests/ComposableArchitectureTests/EffectRunTests.swift

```diff
@@ -122,8 +122,8 @@ final class EffectRunTests: BaseTCATestCase {
   }
 
   #if DEBUG
-    func testRunEscapeFailure() async throws {
```

Comment:
Unrelated warning that popped up. Can move to another PR if we don't merge this anytime soon.